uber/config.py

```diff
@@ -258,15 +259,21 @@ def PRE_CON(self):
         return not self.AT_OR_POST_CON
 
     @property
+    @lru_cache(maxsize=1)
```

Comment:
This function must iterate through the values of the `c.JOB_LOCATIONS` dictionary, rather than the keys; which can be expensive if we do it all the time. Since the value will never change once the config is loaded, I added a caching wrapper around it.